iiasa/ixmp
# Release v3.7.0

This PR prepares the release of version 3.7.0, which will keep our code up to date and solve several issues new users might otherwise experience.



## How to review

- Check that the package is built and published to TestPyPi
- Check that the package can be downloaded, installed, and run
- Check that the README is rendered correctly 

## PR checklist
